cli/src/lib.rs

```diff
@@ -1842,6 +1857,67 @@ fn write_idl(idl: &Idl, out: OutFile) -> Result<()> {
     Ok(())
 }
 
+fn account(
+    cfg_override: &ConfigOverride,
+    account_type: String,
+    address: Pubkey,
+    idl_filepath: Option<String>,
+) -> Result<()> {
+    let (program_name, account_type_name) = account_type.split_once('.').ok_or(anyhow!(
```

Comment:
Any idea for alternative syntax here? I don't love this but I also don't have a better idea. :)

Reply:
Instead of alternate syntax, I've added a check to make sure that this is the only `.` in the name. Would that be alright?